### ISSUE TITLE
Enhancement request: Navigating the "Expansions" menu

### ISSUE THREAD

**Issue by Urutsini** [spacedockapp/spacedock#196]
When looking at individual expansions, when you return to the expansion selection menu, it returns to the top of the list. Is it possible to have the app remember where you were when you selected the expansion without having to scroll back down? (Android)

**Comment by DrNicket** [comment]
Yes this gets tedious quite quickly. Perhaps applying the faction filter would help. It's available as an option, but doesn't work. (Should I make this a new and separate Issue?)

**Comment by Urutsini** [comment]
Filtering Expansions has been open as Issue #207
